Makefile

```diff
@@ -109,10 +109,21 @@ versioninfo:
 	go run scripts/generate_versioninfo/main.go; \
 	go generate cmd/telegraf/telegraf_windows.go; \
 
-.PHONY: telegraf
-telegraf:
+.PHONY: generate
+generate:
+	go generate -run="plugindata/main.go$$" ./plugins/inputs/... ./plugins/outputs/... ./plugins/processors/... ./plugins/aggregators/...
+
+.PHONY: cleanGenerate
+cleanGenerate:
```

Comment:
to bikeshed: our other multi-word make targets are hyphen separated (e.g. check-deps, lint-branch), can we do `clean-generate` or even `generate-clean` so they show up near each other in help output.